docs/localization.md

```diff
@@ -0,0 +1,47 @@
+# Localization
+
+## When should you localize strings?
+
+Any time you add human-readable text to the add-on, whether it is plain text, labels, or text only available to screen-readers, you should localize those strings.
+
+## How to localize strings
+
+First, add the new string to `locales/en-US/strings.properties`. You do NOT need to add strings to other language files.
+
+Note that you must **re-build the add-on** every time you add new strings (the actual build task that processes the strings.properties file is `bundle:locales`, so you need to re-run `npm start`).
+
+On the React side, You have two options for implementing the strings:
```

Comment:
Do we have access to strings from the add-on side? Seems like it gets stashed on the state object.